src/_includes/graphql/store-config.md

```diff
@@ -80,12 +82,18 @@ Attribute | Data Type | Description | Default or example value
 `secure_base_url` | String | The store's fully-qualified secure base URL | `https://magentohost.example.com/`
 `send_friend` | [SendFriendConfiguration](#SendFriendConfiguration) | Email to a Friend configuration | Not applicable
 `show_cms_breadcrumbs` | Int | Determines if a breadcrumb trail appears on all CMS pages in the catalog. Options: `0` (No) or `1` (Yes) | 1
-`store_name` | String | The store's name | `My Store`
+`store_code` | ID | The unique ID of the store view. In the Admin, this is called the Store View Code. When making a GraphQL call, assign this value to the `Store` header to provide the scope | `default`
+`store_group_code` | ID | The unique ID assigned to the store group. In the Admin, this is called the Store Name | `main_website_store`
+`store_group_name` | String | The label assigned to the store group | Main Website Store
+`store_name` | String | The label assigned to the store view | Default Store View
+`store_sort_order` | Int | The store view sort order | 10
 `timezone` | String | The store's time zone | `America/Chicago`
 `title_prefix` | String | A prefix that appears before the title to create a two- or three-part title | null
 `title_separator` | String | Identifies the character that separates the category name and subcategory in the browser title bar | `-`
 `title_suffix` | String | A suffix that appears after the title to create a two-or three part title | null
-`website_id` | Integer | The ID number assigned to the parent website | `1`
+`website_code` | ID | The unique ID for the website | `base`
+`website_id` | Integer | Deprecated. The field should not be used on the storefront. The ID number assigned to the parent website | `1`
+`website_name` | ID | The label assigned to the website | Main Website
```

Comment:
Hi @keharper. I just wanted to double-check that the `website_name` has the `ID` type. From a technical standpoint, it's supposed to be `String`.

Thanks.

Reply:
@rogyar Ugh. That's how it's defined in the schema file. I've notified Cris. Hopefully, we can change this before 2.4.2 ships

Reply:
Good news. We're fixing it.